engines/common/nucleus/particles/menu.html.twig

```diff
@@ -52,7 +52,9 @@
                         {{ SELF.displayTitle(item, gantry) }}
                     </span>
                     {% if (item.children) -%}
```

Comment:
Maybe this instead two nested ifs:

    {% if (item.children) and item.dropdown != 'disabled' -%}